yb-voyager/cmd/live_migration.go

```diff
@@ -57,14 +58,14 @@ func streamChanges() error {
 	if err != nil {
 		return fmt.Errorf("failed to fetch event channel meta info from target : %w", err)
 	}
-	statsReporter := reporter.NewStreamImportStatsReporter()
+	statsReporter = reporter.NewStreamImportStatsReporter()
 	err = statsReporter.Init(tdb, migrationUUID)
 	if err != nil {
 		return fmt.Errorf("failed to initialize stats reporter: %w", err)
 	}
-	defer statsReporter.Finalize()
+	defer finalizeStats()
```

Comment:
can we make - `finalizeStatus` and `updateExportedEventsStats` methods of the `statsReporter` struct

Reply:
No, we can't there is some cyclic dependency issue in having a separate connection to MetaDB in the `streamImportReporter` class. Need to do refactoring of metaDB to some other location like `utils`. to be able to use it everywhere. That's a future task. See this comment for more information https://github.com/yugabyte/yb-voyager/pull/1009#discussion_r1304251485